plugin.video.lacapitv/addon.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<addon id="plugin.video.lacapitv" name="La Capi TV" version="1.0.0" provider-name="Ernesto Bazzano">
+	<requires>
+		<import addon="xbmc.python" version="3.0.0"/>
+		<import addon="script.module.t1mlib" version="4.0.0"/>
+		<import addon="script.module.requests" version="2.22.0"/>
+	</requires>
+	<extension point="xbmc.python.pluginsource" library="default.py">
+		<provides>video</provides>
+	</extension>
+	<extension point="xbmc.addon.metadata">
+		<platform>all</platform>
+		<summary lang="en_GB">Interspace television</summary>
+		<description lang="en_GB">An intergalactic streaming service where you can find La Capi animations among many other series and small animation experiments.
+</description>
+		<summary lang="es_AR">Una televisión interespacial</summary>
+		<description lang="es_AR">Un servicio de streaming intergaláctico donde podés encontrar animaciones de La Capi entre tantas otras series y pequeños experimentos de animación.
+</description>
+		<license>GPL-3.0-or-later</license>
+		<language>es en</language>
```

Comment:
this refers to the language of the content. Does it provide english videos or only in spanish? If only spanish please keep it as `es`. This tag allows filtering the addons per content language in the kodi ui